integration_testing/features/learner/learner-take-exam.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Learner take exam
+  Learner can access the exam that has been assigned by coach
+
+  Background:
+    Given I am signed in to Kolibri as a Learner user
+      And I am on *learn > classes* page
+      And I see an exam not started
+      And I am on the *learn > classes > exam* page
+
+  Scenario: Learner take an exam after it has been assigned
+    When I fill out all questions with answers
+    When I click *Submit exam* button
+    Then The submit exam modal show up
+    When I click *Submit exam* button
+    Then I see exam completed
```

Comment:
@mrpau-dev very minor comment - make sure to set your text editor to always include a newline at the end of the file.

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

(most text editors have a setting to enforce this on save)

Reply:
Thanks @indirectlylit, I myself am often guilty as charged for trimming that, instead of leaving the newline... 😕!

Reply:
Note it doesn't mean that you need to leave a blank line. See e.g.

https://github.com/learningequality/kolibri/commit/abe0a15e43ecb62cd591d616bda3432c32a51c71#diff-c00eb2f13f7f14aac3118a26c0063ea8R9

vs 

https://github.com/learningequality/kolibri/commit/abe0a15e43ecb62cd591d616bda3432c32a51c71#diff-ecc2c18a94f5c8a056ff447696c558dbR48